jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/RuntimeDataServiceImpl.java

```diff
@@ -811,6 +811,17 @@ public UserTaskInstanceDesc getTaskById(Long taskId) {
         return null;
 	}
 
+    @Override
+    public UserTaskInstanceDesc getTaskByIdWithSLA(Long taskId) {
```

Comment:
@nmirasch perhaps to the same here as in the kie-server, same method name with boolean withSLA? That way you could reuse the method again with a default of false.

Reply:
@cristianonicolai updated, Thanks!